Owl.install

```diff
@@ -0,0 +1,6 @@
+etc: [
+  "setup.ml"
+  "setup.data"
+  "setup.log"
+  "_oasis_remove_.ml"
+]
```

Comment:
@struktured why is this needed?  shouldn't `install` below just call out to `ocaml setup.ml -install`?